### ISSUE TITLE
Implement early stopping in NumPy dense neural network

### ISSUE THREAD

**Issue by AditMeh** [AditMeh/deep-learning-from-scratch#2]
Currently, I am able to compute test loss, train loss, train accuracy, and test accuracy. Take these python lists and implement a function that looks at the previous _n_ test loss values to check for increasing test loss. Stop training if test loss has been increasing for the past _n_ epochs.
